packages/AutoTDD-Core.package/ATDDTestingEntity.class/class/neutralStatus.st

```diff
@@ -0,0 +1,4 @@
+constants
+neutralStatus
+
+	^ #pending
```

Comment:
Shouldn't all the status be a string?

```suggestion
	^ 'pending'
```

Also: Do we use this method anywhere?

Reply:
Yes, there are more places we should even use it, and it is a generalized status so the symbol remains